packages/pwa-buildpack/lib/WebpackTools/configureWebpack/getModuleRules.js

```diff
@@ -96,13 +96,21 @@ getModuleRules.js = async ({
  * @returns Rule object for Webpack `module` configuration which parses
  *   CSS files
  */
-getModuleRules.css = async ({ paths, hasFlag }) => ({
+getModuleRules.css = async ({ mode, paths, hasFlag }) => ({
     test: /\.css$/,
     oneOf: [
         {
             test: [paths.src, ...hasFlag('cssModules')],
             use: [
-                'style-loader',
+                {
```

Comment:
@jcalcaben you may wanna update the docs with this new webpack config for style tags. A quick search for `style-loader` showed a couple of doc files mentioning it.

Reply:
@revanth0212 I've updated the docs at css-modules configuration.

https://github.com/magento/pwa-studio/pull/2906/commits/47731764e7ff5103f4a5ada08e64b4787b715184